watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -334,7 +338,7 @@ def calculate_operating_pressure(
     t.brine[0].pressure_osm_phase
     # solve state block
     results = solve_indexed_blocks(solver, [t.brine])
-    assert_optimal_termination(results)
+    check_optimal_termination(results)
```

Comment:
My one comment is that I'm still not a huge fan of these `check_optimal_terminations` where we do nothing with the result.

This is a no-op, so it might confuse someone reading this code as to why it's here in the first place.